tests/acceptance/features/apiLocks/lockFiles.feature

```diff
@@ -312,7 +312,7 @@ Feature: lock files
       | new              |
       | spaces           |
 
-  @issue-7638
+    @issue-7638
```

Comment:
```suggestion
  @issue-7638
```

The indentation change causes the linter to fail this feature file, and so the tests do not even run.